src/form/bf_mx.jl

```diff
@@ -95,8 +95,8 @@ function variable_tp_branch_series_current_prod_hermitian(pm::_PMs.GenericPowerM
         y_to_mag = abs.(branch["g_to"].values + im* branch["b_to"].values)
 
         smax = branch["rate_a"].values
-        cmaxfr = smax./vmin_fr + vmax_fr.*y_fr_mag
-        cmaxto = smax./vmin_to + vmax_to.*y_to_mag
+        cmaxfr = smax./vmin_fr + y_fr_mag*vmax_fr
+        cmaxto = smax./vmin_to + y_to_mag*vmax_to
```

Comment:
It's not immediately clear to me that this is correct.  Did you do a derivation?

Reply:
Yes, I looked at these valid bounds in detail when I worked on the bf rewrite. The current is given by
`I = Y.V + conj(S./V)`

This implies a valid bound on the magnitude of I, |I|, as a function of Smax, Vmax and Vmin
`|I| <= |Y|.|V| + |S|./|V| <= |Y|.Vmax + Smax./Vmin`

In scalar form it is clearer that this is correct. 
`I[c] = sum(Y[c,d]V[d] over d) + conj(S[c]./V[c])`

The valid bound is arrived at by repeatedly applying the triangle inequality.
`|I[c]| = |sum(Y[c,d]V[d] over d) + conj(S[c]./V[c])| <= |sum(Y[c,d]V[d] over d)| + |conj(S[c]./V[c])|  <= sum(|Y[c,d]|.|V[d]| over d) + |S[c]|./|V[c]|`

Reply:
Perhaps it is possible to derive a tighter valid bound by rewriting the equality first, but at least this is a valid extension to the matrix branch shunts.

Reply:
valid is sufficient, thanks for the explanation.